### DISCUSSION TITLE
Couldn't update code server

### DISCUSSION THREAD

**Discussion by ghost** [coder/code-server#2736] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/27614506/108105132-55275f80-70b4-11eb-94b9-db2f06621e13.png)
![image](https://user-images.githubusercontent.com/27614506/108105165-5f495e00-70b4-11eb-8ab2-80082d46afe5.png)

I tried updating code-server by these commands. but while checking the vs code it's still un updated.

**Comment by nooobcoder** [comment, marked as answer]
@beast076  You already have the latest version of code-server. Here is the screenshot from my interface. I updated it 3 days back and there aren't any updated. Latest version is **3.9.0** as of now.
 ![image](https://user-images.githubusercontent.com/50350828/108225699-2d8ed080-7162-11eb-8d3c-442a1b311e67.png)

Please mark the discussion thread as answered if your doubt has been resolved. I don't think there is any further discussion to be made regarding this.

**Comment by ghost** [comment]
okek. thanks for answering :)

**Comment by ghost** [comment]
oh btw can i ask which theme you using?

**Reply by nooobcoder** [reply]
@beast076 I use Shades of Purple, it is the best theme I have tried in my life ever. Also check out Bear theme i am attaching links to the vs-code marketplace below
https://marketplace.visualstudio.com/items?itemName=ahmadawais.shades-of-purple
https://marketplace.visualstudio.com/items?itemName=dahong.theme-bear
